xclim/temperature.py

```diff
@@ -7,20 +7,21 @@
 include a number of functionalities, such as input validation, unit conversion, output meta-data handling,
 and missing value masking.
 
-The concept followed here is to define UnivariateIndicator subclasses for each input variable, then create instances
+The concept followed here is to define Indicator subclasses for each input variable, then create instances
 for each indicator.
 
 """
 
 from . import checks
 from . import indices as _ind
-from .utils import UnivariateIndicator
+from .utils import Indicator
 
+Indicator = BivariateIndicator = Indicator
```

Comment:
What's happening with the assignments here?

Reply:
This is a leftover. Should be deleted.